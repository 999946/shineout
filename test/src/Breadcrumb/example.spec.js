import exampleTest from '../../example'

exampleTest('Breadcrumb')
