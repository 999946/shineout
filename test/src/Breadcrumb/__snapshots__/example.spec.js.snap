// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Breadcrumb[site/pages/components/Breadcrumb/example-1-base.js] 1`] = `
<div>
  <span>
    <a>
       
      Home
    </a>
    <span>
      /
    </span>
  </span>
  <span>
    <a>
      Button
    </a>
    <span>
      /
    </span>
  </span>
  <span>
    <a>
       
      Self
    </a>
  </span>
</div>
`;

exports[`Snapshot test: Breadcrumb[site/pages/components/Breadcrumb/example-2-separator.js] 1`] = `
<div>
  <div>
    <span>
      <a>
         
        Home
      </a>
      <span>
        |
      </span>
    </span>
    <span>
      <b>
        Self
      </b>
    </span>
  </div>
  <div>
    <span>
      <a>
         
        Home
      </a>
      <span>
        <span>
          ~
        </span>
      </span>
    </span>
    <span>
      <b>
        Self
      </b>
    </span>
  </div>
</div>
`;

exports[`Snapshot test: Breadcrumb[site/pages/components/Breadcrumb/example-3-icon.js] 1`] = `
<div>
  <span>
    <a>
      <i />
       
      Home
    </a>
    <span>
      /
    </span>
  </span>
  <span>
    <b>
      Menu
    </b>
    <span>
      /
    </span>
  </span>
  <span>
    <a>
       
      Self
    </a>
  </span>
</div>
`;
